aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -202,6 +207,95 @@ For vertical lists:
   <NameValueListReadMoreExample />
 </Example>
 
+## Name-Value Pair with Empty Value
+
+If the value of a name-value pair is empty, use `--` in place of where a value would appear. This visual indicator aids the user in identifying fields that do not have values without increasing cognitive load.
```

Comment:
Thinking this would treat the accessibility aspect as more of a first-class consideration.

```suggestion
If the value of a name-value pair is empty, use `--` in place of where a value would appear. This visual indicator aids the user in identifying fields that do not have values without increasing cognitive load.

Add an `a11yTitle="No data"` to the Text component that would otherwise render the data to ensure users of assistive technology are informed of the empty state.
```

Reply:
Done.